data/core.yaml

```
@@ -1992,6 +1997,9 @@ en:
       remove_tag:
         title: Remove the tag
         annotation: Removed tag.
+      remove_named_tag:
+        title: Remove the '{tag}' tag
+        annotation: Removed '{tag}' tag.
```

Use double quotation marks for consistency with this validator rule’s `reference` and with other validator rules such as `tag_as_not`:

```suggestion
        title: 'Remove the "{tag}" tag'
        annotation: 'Removed "{tag}" tag.'
```

---

good idea